wavepool/templates/wavepool/newspost.html

```diff
@@ -34,4 +38,5 @@ <h1 id="newspost-title">{{ newspost.title }}<span class="pubdate">{{newspost.pub
 			Tags: {% for tag in newspost.tags %}{{tag}}{% if not forloop.last %},{% endif %} {% endfor %}
```

Comment:
I now realize that this is referring to the tab on top of the page, not the title display. I apologize, but the solution is easy to address with get_context_data